styles/themes/colors.css

```diff
@@ -28,12 +28,10 @@
   --dot: #bbb;
   --dot-active: #717171;
 
-  --footer: #292c2f;
-  --footer-white: #f5f5f5;
-  --footer-icons: #b194a8;
-  --footer-name: #a5a9af;
-  --footer-link: #5383d3;
-  --footer-about: #92999f;
+  --footer-text-primary: #a5a9af;
+  --footer-text-secondary: #f5f5f5;
+  --footer-bg: #292c2f;
+  --footer-icon: #b194a8;
```

Comment:
I would strongly consider to reduce the amount of colours. A lot of them are very similar but vary only slightly. Pick 1 kind of white for instance for the footer text and the article background etc. 

Usually I go with a colour setup that looks like this ( picking random colours for demo purpose ): 

```css
:root {
    /* base colors */
    --primary-color: white;
    --secondary-color: black;
    --tertiary-color: grey;
    
   /* accent colors */
    --primary-accent-color: pink;
    --secondary-accent-color: lightpink;
    
    /* fonts */
    --primary-font-color: var(--primary-color);
    --secondary-font-color: var(--tertiary-color);
    
  --primary-accent-font-color: var(--primary-accent-color);
  --secondary-accent-font-color: var(--secondary-accent-color);
}
```

Note here how font colors use the base and accent colours we defined. For your footer for instance, you would then just say: 

```css
.footer {
   color: var(--secondary-font-color);
}

.footer-link-list {
    color: var(--primary-font-color);
}
```

etc. This way, when you decide you want to update 1 shade of color ( for ex your greys ), you only need to change the hexcode in 1 line. 

Using a limited number of colors also brings cohesion to your site, much like with the alignments / invisible boxes I explained before